addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/UsernameIdorScanRule.java

```diff
@@ -48,7 +48,7 @@
     private static final String ADMIN = "Admin";
     private static final String ADMIN_2 = "admin";
 
-    public static final List<String> DEFAULT_USERNAMES = Arrays.asList(ADMIN, ADMIN_2);
+    private static final List<String> DEFAULT_USERNAMES = Arrays.asList(ADMIN, ADMIN_2);
```

Comment:
Why not make it unmodifiable? (The collection can still be changed with the getter as is now.)

Reply:
In which case it can be left public.